### PR TITLE
Bump version to 1.1, add github_changelog_generator

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: lib/dep-selector-libgecode/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module DepSelectorLibgecode
-  VERSION = "1.0.2"
+  VERSION = "1.1"
```
Comment (danielsdeleo) [inline lib/dep-selector-libgecode/version.rb]: `1.1.0` ?

**Comment by schisamo** [comment]
I don't see any entries in the generated CHANGELOG for this version. What exactly constitutes the version bump in this PR?

**Comment by jkeiser** [comment]
@schisamo https://github.com/chef/dep-selector-libgecode/blob/jk/version-bump/CHANGELOG.md, 1.1.

@dan will fix.

**Comment by schisamo** [comment]
:+1:

**Comment by jkeiser** [comment]
@schisamo @danielsdeleo how does this get tested before release? Is it included in anything already?

**Pull request merged**